x-pack/plugins/cases/server/client/cases/bulk_get.ts

```diff
@@ -74,33 +71,29 @@ export const bulkGet = async <Field extends keyof Case = keyof Case>(
         operation: Operations.bulkGetCases,
       });
 
-    const requestForTotals = ['totalComment', 'totalAlerts'].some(
-      (totalKey) => !fields || fields.includes(totalKey)
-    );
-
-    const commentTotals = requestForTotals
-      ? await attachmentService.getter.getCaseCommentStats({
-          caseIds: authorizedCases.map((theCase) => theCase.id),
-        })
-      : new Map();
+    const commentTotals = await attachmentService.getter.getCaseCommentStats({
+      caseIds: authorizedCases.map((theCase) => theCase.id),
+    });
 
     const flattenedCases = authorizedCases.map((theCase) => {
-      const { alerts, userComments } = commentTotals.get(theCase.id) ?? {
+      const { userComments } = commentTotals.get(theCase.id) ?? {
```

Comment:
Why is the alerts logic being removed?

Reply:
As it is not used anywhere at the moment I thought to keep the fields returned as narrow as possible. Do you think we should return the `totalAlerts` variable?

Reply:
> Do you think we should return the totalAlerts variable?

Nah, I was really just looking for the why 👍